src/en/caas.md

```diff
@@ -0,0 +1,192 @@
+Title: CAAS and Juju
```

Comment:
Generally, CAAS should be "Kubernetes" as per Mark's latest thinking on terminology.